exoctk/contam_visibility/miniTools.py

```diff
@@ -0,0 +1,363 @@
+""" The contamVerify mini tool will be a companion to ExoCTK's Contamination
+Overlap tool, as it will visualize the Contaminaton Bokeh plots on the website.
+
+Functions are:
+plotTemps    - Plots the temperatures of stars according to color.
+traceLengths - Fine-tunes the trace lengths in the plot.
+contamVerify - The main mini tool. Outputs a .pdf file with one or more figures
+               showing the FOV in the science frame according to the input
+               Aperture Position Angle(s) it is fed.
+
+Author(s)
+---------
+Jennifer V. Medina, 2020
+"""
+import os
```

Comment:
I think PEP8 prefers to list imports in alphabetical order (in each import section)?